config/services.yaml

```diff
@@ -57,3 +59,8 @@ services:
                connection: default
         calls:
             - [ setAnnotationReader, [ "@annotation_reader" ] ]
+
```

Comment:
Für was benötigen wir einen Keycloak authenticaor?

Reply:
Der KeycloakAuthenticator ist nicht neu. Er wurde hier eingeführt: https://github.com/H2-invent/open-datenschutzcenter/commit/b0e7a2028052accedcfe235c2475d715c0242f2e

Neu sind nur die beiden Parameter:
```
App\Security\KeycloakAuthenticator:
        arguments:
            $groupApiUserId: '%group_api_user_id%'
            $groupApiRole: '%group_api_role%'
```
Bei mir sehen die zum Beispiel so aus im env.local:
```
GROUP_API_USER_ID=user_identifier
GROUP_API_ROLE=585127
```
Mit dem ersten Parameter wählst du eine OIDC User Claim, der als UserID an die API gesendet wird. Der zweite gibt eine Rollen ID an. Wir brauchen die Rollen ID, weil wir die Anforderung haben, dass nur spezielle User mit der Rolle ODC Redakteur als Team Mitglieder hinzugefügt werden. Wer die Rolle nicht hat, der wird keinem Team zugeteilt.

Allgemein könnten wir natürlich auch über die aktuell verwendete OIDC Libs sprechen. Symfony unterstützt OIDC eigentlich von alleine seit 6.3. https://symfony.com/blog/new-in-symfony-6-3-openid-connect-token-handler
Da sehe ich schon Potential den Keycloakauthenticator komplett auszutauschen.

Aber das ist vielleicht out of scope für diesen PR?